src/zcash/address/unified.h

```diff
@@ -27,6 +27,17 @@ enum class ReceiverType: uint32_t {
     Orchard = 0x03
 };
 
+/**
+ * An enumeration of the fund pools for which a transaction may produce outputs.
+ * It is sorted in descending preference order, so that when iterating over a
+ * set of output pools the most-preferred pool is selected first.
+ */
+enum class OutputPool {
```

Comment:
Why is this in `unified.h`? It is not specific to unified addresses.

Reply:
We need it here now because we need it in `ZcashdUnifiedFullViewingKey::GetChangeAddress`.